crates/virtio-queue/src/chain.rs

```diff
@@ -104,10 +104,9 @@ where
             return Err(Error::InvalidIndirectDescriptor);
         }
 
-        // Check the target indirect descriptor table is correctly aligned.
-        if desc.addr().raw_value() & (VRING_DESC_ALIGN_SIZE as u64 - 1) != 0
-            || desc.len() & (VRING_DESC_ALIGN_SIZE - 1) != 0
-        {
+        // Alignment requirements for vring elements start from virtio 1.0,
```

Comment:
Can you also add an entry in the virtio-queue changelog for this? It's not an API change, but it's an important bug fix.

Reply:
Sorry, I am not familiar with the release process, I check the commit history of CHANGELOG.md, the changelog will be updated when new release comes, this PR should be an issue fix for next release?

Reply:
Yes, we can update the changelog either right before the release with all new changes, or incrementally in each PR. We are trying to go with the second option, that's why we added it as a check in the PR template. You should just add a Fixed section in the changelog and one entry for your fix. Thanks!

Reply:
OK, I have added this change into the changelog.